### PR TITLE
* refactored significantBits

### PR DESCRIPTION
**Pull request by adamnemecek**

### CODE REVIEW

**Comment by Jitsusama** [comment]
Wow, this library is starting to go on a major diet! 😄 

It looks good to me. Am I right in guessing that this change passed tests as well?

File: Sources/UInt128.swift [hunk 1]
```diff
@@ -45,24 +45,7 @@ public struct UInt128 {
...
-        }
-
-        // Walk significant bits by shifting right until all bits are equal to 0.
-        while bitsToWalk > 0 {
-            bitsToWalk >>= 1
-            significantBits += 1
-        }
-
-        return significantBits
+        return  UInt128(UInt128.bitWidth - leadingZeroBitCount)
```
Comment (Jitsusama) [inline Sources/UInt128.swift]: There are 2 spaces in between the return and the UInt128 call, please reduce this to 1 space.

**Comment by adamnemecek** [comment]
Iteration is against my religion :-).

Yeah it does. And fixed the ws.

**Comment by Jitsusama** [comment]
Thanks so much!

**Pull request merged**